tensorflow_recommenders_addons/dynamic_embedding/core/BUILD

```diff
@@ -29,7 +29,10 @@ custom_op_library(
         "utils/utils.h",
         "utils/types.h",
     ] + glob(["kernels/lookup_impl/lookup_table_op_gpu*"])),
-    deps = ["//tensorflow_recommenders_addons/dynamic_embedding/core/lib/cuckoo:cuckoohash"],
+    deps = [
+        "//tensorflow_recommenders_addons/dynamic_embedding/core/lib/cuckoo:cuckoohash",
+        "//tensorflow_recommenders_addons/dynamic_embedding/core/lib/hadoop_file_system",
```

Comment:
If the hdfs so the file is too big, we'd best to make it a user-responsible-so. The TFRA just invokes and depends on the header of HDFS

Reply:
If we use the original hdfs, we need to implement a series of stream and buffer methods, I think it would be better to reuse the tf native interface (e.g., RandomAccessInputStream, BufferedInputStream)

Reply:
The current dependency on `hadoop_file_system` is `libtensorflow_framework` and hdfs header files, because libtensorflow_framework contains hdfs so.

Reply:
Thank you for your clarification.